engine/crash/src/script_crash.cpp

```diff
@@ -13,7 +13,7 @@
 // specific language governing permissions and limitations under the License.
 
 #include <assert.h>
-#include <extension/extension.h>
+#include <extension/extension.hpp>
```

Comment:
It just became too brittle, and I got C vs C++ compilation issues.
Better to be correct: If you're using C++ api, use the .hpp header.